commands/proposal_submission.go

```diff
@@ -1858,6 +1870,72 @@ func checkLiquidityFeeSettings(config *protoTypes.LiquidityFeeSettings, parent s
 	return errs
 }
 
+func checkCompositePriceConfiguration(config *protoTypes.CompositePriceConfiguration, parent string) Errors {
+	errs := NewErrors()
+	if config == nil {
+		errs.AddForProperty(parent, ErrIsNotValid)
+		return errs
+	}
+	if config.DecayPower < 0 || config.DecayPower > 3 {
```

Comment:
Is there a reason we're not using a `uint` for the decay power?

Reply:
not really, I'll change to uint